src/test/java/com/twilio/oai/PathUtilsTest.java

```diff
@@ -116,4 +116,22 @@ public void testIsInstanceOperation() {
         instanceCo.path = "http://twilio.com/Account/{Sid}/History/{TestInteger}";
         assertTrue(PathUtils.isInstanceOperation(instanceCo));
     }
+
```

Comment:
can we also have a test by adding dependents info in our [test spec](https://github.com/twilio/twilio-oai-generator/blob/main/examples/spec/twilio_api_v2010.yaml), and validating the result in a test.

Reply:
Added.